### ISSUE TITLE
Attribute style in HTML example should be rendered but removed from code snippet

### ISSUE THREAD

**Issue by vitto** [vitto/a-pollo#7]
The meaning of insert style attribute in HTML example is to prevent unwanted default behavior to ease widget visualization inside style guide only, so it should be removed from code snippet.

```
/*@pollo
    @name:  Club
    @auth:  [Vittorio Vittori](http://vit.to)
    @category:  News
    @date:  2016-03-21T12:27:33+01:00
    @text:  This widget has position absolute positioning and it's not nice inside the style guide
    @html:  <span class="news-club" style="position:static;">
                <i class="serie-a serie-a--inter"></i>
            </span>
*/
```

There are cases where `style` attribute should be shown, maybe could be the case to add a new doc tag, something like `@style: true` (`false` by default).

This could be useful for background images of styled avatars.

**Comment by fain182** [comment]
So this widget works only inside a particular context, maybe is better to add the context in the example?

I mean something like:

```
<div class="news-container">
     <span class="news-club" style="position:static;">
          <i class="serie-a serie-a--inter"></i>
     </span>
</div>
```

**Comment by vitto** [comment]
I mean there are mainly two contexts:

---

###### Case 1

The style should be **removed** because it's added just to prevent weird behavior inside the style guide:

```
<span class="news-club" style="position:static;">
    <i class="serie-a serie-a--inter"></i>
</span>
```

So `style="position:static;"` shouldn't be copied in the clipboard when the developer clicks on the snippet.

---

###### Case 2

The style should be **kept** because the developer should "custom" the style attribute with dynamic data:

```
<span class="avatar" style="background-image:url(http://placehold.it/32x32);"></span>
```

So `style="background-image:url(http://placehold.it/32x32);"` should be copied in the clipboard when the developer clicks on the snippet.

**Comment by vitto** [comment]
Also, I can just force the first level element of the HTML example container to be in `position: static;`.

**Comment by vitto** [comment]
Resolved with commit [4dee482](https://github.com/vitto/a-pollo/commit/4dee4829507e9f7a1f4c22035a63cc128e42e345).
